spec/models/candidate_interface/other_qualification_form_spec.rb

```diff
@@ -57,6 +57,7 @@
   describe '.build_all_from_application' do
     let(:application_form) do
       create(:application_form) do |form|
+        form.application_qualifications.create(level: 'other')
```

Comment:
Just wondering what happens if there is a dead heat for `created_at` between this and the one below. It could be avoided by explicitly specifying different `created_at` values.

Reply:
Ah yes! I thought I did that, but between things and my brain not working very well I hadn't done this. Will update, thank you!

Reply:
Updated now as you suggested, with specifying `created_at` values.